src/omotes_orchestrator/postgres_interface.py

```diff
@@ -13,27 +13,22 @@
 
 LOGGER = logging.getLogger("omotes_orchestrator")
 
-session_factory = orm.sessionmaker()
+session_factory = orm.sessionmaker(expire_on_commit=False)
```

Comment:
This prevents us from having to 'expunge' any results after the session ended. They are automatically 'detached' with this flag.  https://docs.sqlalchemy.org/en/20/orm/session_state_management.html